src/ui/Combobox/ComboboxInput.ts

```diff
@@ -48,13 +48,11 @@ export class ComboboxInput {
 
   private addAccessibilityAttributes() {
     const listboxId = `${this.combobox.id}-listbox`;
-    this.element.setAttribute('role', 'combobox');
-    this.element.setAttribute('aria-owns', listboxId);
-    this.element.setAttribute('aria-haspopup', 'listbox');
-    this.element.setAttribute('aria-autocomplete', 'list');
-
+    this.inputElement.setAttribute('role', 'combobox');
+    this.inputElement.setAttribute('aria-owns', listboxId);
+    this.inputElement.setAttribute('aria-haspopup', 'listbox');
+    this.inputElement.setAttribute('aria-autocomplete', 'list');
     this.inputElement.setAttribute('id', `${this.combobox.id}-input`);
-    this.inputElement.setAttribute('aria-controls', listboxId);
```

Comment:
This attribute was removed because it appears to be redundant and wasn't included in Deque's demo.